changelog.txt

```diff
@@ -8,6 +8,7 @@
 * Enhancement - [Transparency] CLI command for viewing tracking data for your store. #30010
 * Enhancement - All settings pages can now be extended consistently with new sections and settings. Also, unit tests have been added. #27684
 * Enhancement - Set checkout fields value with the default defined value where form is not presented to the user. #29820
+* Enhancement - Update `locale-info.php` to contain all the countries, currency specs and locale information. #30216
```

Comment:
Can you please remove this change as we usually add the changelogs from our side.

Reply:
Done.